Source/WebCore/animation/KeyframeEffect.cpp

```diff
@@ -128,8 +128,11 @@ static inline CSSPropertyID IDLAttributeNameToAnimationPropertyName(const AtomSt
     // 3. If attribute is the string "cssOffset", then return an animation property representing the CSS offset property.
     if (idlAttributeName == "cssOffset"_s)
         return CSSPropertyOffset;
+    // 4. If the attribute is the string "fontStretch" return the CSS offset property that it aliases.
```

Comment:
The numbering comes from the Web Animations spec, so you can't increase it here. Also, you copied and pasted "offset" here.
```suggestion
    // If the attribute is the string "fontStretch" return the "font-width" property that it aliases.
```